react/features/mobile/navigation/components/welcome/components/WelcomePageTabs.js

```diff
@@ -68,7 +68,7 @@ const WelcomePageTabs = ({ disabled, onListContainerPress, onSettingsScreenFocus
 
     return (
         <WelcomePage.Navigator
-            initialRouteName = { screen.welcome.tabs.recent }
```

Comment:
Not needed because by default this will be the first screen every time we enter the app or exit a conference room.